### PR TITLE
new metamask banner

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

This PR is aimed at retiring the metamask banner on the left side of the explorer & bounty details page.
It's to be replaced with a tag like element which has 
- a green dot if connected to the right network
- a red dot with relevant text telling them why it's not connected to the right network + 
   additinal info on hover + 
   a red border line pops up below the navbar to bring attention to the user that they are on the wrong 
   network.

**Right network :** 

screen shot 2018-07-01 at 9 58 17 pm

**Wrong network / Metmask Locked / Not Installed :** 

screen shot 2018-07-01 at 9 48 06 pm

screen shot 2018-07-01 at 9 48 15 pm

As of now -> I've retired the old banner in all  pages except ens (cause that is super messy)
If that's removed -> we can remove a lot of code along with it. (need help @SaptakS )

Need help testing this across all pages before merge.

@owocki @mbeacom @SaptakS Could you test it out and tell me if we can make it better ?

Also  @PixelantDesign , wanted the same text on the tag on the red line when user is on the wront network. It means duplicating info which is already available on the above tag. Your thoughts ?

screen shot 2018-07-01 at 9 54 32 pm

^ The red line would have some content -> "Connected to Wrong Network" / etc )

Left to do (in another PR): 
- [x] update ens page to use new tag
- [x] remove old code 
- [x] banner to show up only in "non-marketing" pages

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

### CODE REVIEW

File: app/assets/v2/js/shared.js [hunk 1]
```diff
@@ -599,6 +599,64 @@ var trigger_sidebar_web3 = function(network) {
...
+      $('.navbar-network i').addClass('red');
+      $('.navbar-network i').removeClass('green');
+      $('#navbar-network-banner').addClass('network-banner--warning');
+
+      if ($('.ui-tooltip.ui-corner-all.ui-widget-shadow.ui-widget.ui-widget-content').length == 0) {
+        $('.navbar-network').attr('title', tooltip_info);
+      }
+    }
+  } else { // Staging
+    if (network == 'rinkeby') {
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/assets/v2/js/shared.js]: But this makes it more clear :|

**Comment by owocki** [comment]
very excited about this! way cleaner

1. its hard to click the link on the tooltip http://bits.owocki.com/3t0O042m020K/Screen%20Recording%202018-07-01%20at%2009.07%20PM.gif
2. new element should be aligned better relative to logo http://bits.owocki.com/2y3r3I1b0Y00/Screen%20Shot%202018-07-01%20at%209.08.16%20PM.png
3. logo disappears on landing page when u scroll down and this element replaces it http://bits.owocki.com/3c2h0K3D1g0H/Screen%20Recording%202018-07-01%20at%2009.08%20PM.gif

**Comment by thelostone-mc** [comment]
1. have to think of a better way ! The tooltip disappears the moment you go out of focus  🤔
@SaptakS any idea / do we just remove the link ?  


2 & 3 have been taken care ^_^

screen shot 2018-07-02 at 11 58 26 am

Hat-tip to @owocki  being a super neat tester :P

**Comment by thelostone-mc** [comment]
@mbeacom @SaptakS  how do I reach this template on the screen ? 
https://github.com/gitcoinco/web/blob/master/app/dashboard/templates/ens/account.html

File: app/assets/v2/js/shared.js [hunk 2]
```diff
@@ -544,62 +544,84 @@ var randomElement = function(array) {
...
+      if (!network) {
+        info = 'Web3 disabled. Please install <a href="https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral" target="_blank" rel="noopener noreferrer">Metamask</a>';
+        $('#current-network').text('Metamask Not Enabled');
+        $('#navbar-network-banner').html(info);
+      } else if (network == 'locked') {
+        info = 'Web3 locked. Please unlock <a href="https://metamask.io/?utm_source=gitcoin.co&utm_medium=referral" target="_blank" rel="noopener noreferrer">Metamask</a>';
+        $('#current-network').text('Metamask Locked');
+        $('#navbar-network-banner').html(info);
+      } else {
+        info = 'Connect to Mainnet via Metamask';
```
Comment (mbeacom) [inline app/assets/v2/js/shared.js]: Should we wrap the copy throughout with `gettext()` ?

File: app/dashboard/templates/dashboard.html [hunk 3]
```diff
@@ -34,7 +34,6 @@
     {% include '_dashboard/ftux.html' %}
     <div id="dashboard-content" class="row no-gutter">
       <div class="col-12 col-lg-3 col-xl-2 invisible" id="sidebar_container" style="background: #F2F6F9;">
-        {% include 'shared/current_balance.html' %}
```
Comment (mbeacom) [inline app/dashboard/templates/dashboard.html]: 🙌

File: app/dashboard/templates/ens/account.html [hunk 4]
```diff
@@ -1,28 +0,0 @@
-{% extends 'ens/base.html' %}
```
Comment (mbeacom) [inline app/dashboard/templates/ens/account.html]: 🔥

File: app/retail/templates/shared/nav.html [hunk 5]
```diff
@@ -23,9 +23,15 @@
     {% if request.path == '/ios' %}
       <img id="logo" src="{% static "v2/images/ios/gitcoin.svg" %}" data-hover="{% static "v2/images/ios/gitcoin_hover.gif" %}" height="50" />
     {% else %}
-      <img id="logo" src="{% static "v2/images/logo_large.png" %}" data-hover="{% static "v2/images/logo_med_hover.gif" %}" width="180" height="60" />
+      <img id="logo" src="{% static "v2/images/logo_large.png" %}" data-hover="{% static "v2/images/logo_med_hover.gif" %}" width="150" height="50" />
     {% endif %}
   </a>
+  {% if not request.path|is_in_list:"/,/contributor,/about,/mission,/results,/slack" %}
```
Comment (mbeacom) [inline app/retail/templates/shared/nav.html]: 🙌

Comment (thelostone-mc) [reply app/retail/templates/shared/nav.html]: Learnt from the best after all :D

**Comment by thelostone-mc** [comment]
@mbeacom all changes done 👍

**Pull request merged**